docs/remote/troubleshooting.md

```diff
@@ -195,14 +195,33 @@ SSH can be strict about file permissions and if they are set incorrectly, you ma
 
 ### Local SSH file and folder permissions
 
+#### macOS / Linux
```

Comment:
H4s don't render very well on our site so I've been changing platform headers to bold text.

**macOS / Linux:**
 
I can update after the merge.